Header h
int32 ac_id
float64 airspeed        # airspeed (in m/s)
float64 groundspeed     # groundspeed (in m/s)
float64 vertical_speed  # vertical speed (in m/s)
